pytest_html/plugin.py

```diff
@@ -225,7 +226,9 @@ def append_log_html(self, report, additional_html):
             for header, content in report.sections:
                 log.append(' {0} '.format(header).center(80, '-'))
                 log.append(html.br())
-                log.append(content)
+                content = Ansi2HTMLConverter(inline=True).convert(content,
+                                                                  full=False)
```

Comment:
This will add inline styles, which won't work if CSP is enforced. Could you try calling `get_styles`(from ansi2html.style import get_styles) to append the generated styles to the existing style.css, or create a separate ansi.css that's linked. If the user specifies `--self-contained` then we could append the styles to the header, which should be better than repeating styles with `inline=True`.